articles/getting-started-with-matlab/README.md

```diff
@@ -0,0 +1,207 @@
+```
+
+#### M-files
+Entering commands at MATLAB prompt is not practical when the problems will be complicated. The solution will be using M-files which is user-defined commands that normally have input & output and the script will be executed in sequence when it is called.
+
+It is saved with the extension ".m".
+
+Now let's create a function. First of all, you need to press on *New Script*:
+
+![](/engineering-education/getting-started-with-matlab/1.PNG)
+
+Second, press on *New* then *Function*:
+
+![](/engineering-education/getting-started-with-matlab/2.PNG)
+
+It is that easy! Now let's create a circle area function by:
+```matlab
+function [area] = areafunction(r)
+%UNTITLED3 Summary of this function goes here
+%   Detailed explanation goes here
+area = pi*r*r;
+end
+```
+
+After that, you have to save the function with the extension ".m" as we mentioned above.
+Finally, write `areafunction(2)` in the command window and the result will be:
+```matlab
+ans =
+
+   12.5664
+```
+
+#### Plotting
+If you are familiar with the Matplotlib plotting library for Python programming language, it will be very easy for you to follow this section. However, we will explain it simply as possible.
+
+To plot any graph we need to define the range of *X* and *Y* values. Then to call the plot command which is `plot(X,Y)`.
+
+Now let's move on to a complex example. We are going to create the following figure:
+
+![sin and cos waves](/engineering-education/getting-started-with-matlab/sin-cos.jpg)
+
+First of all, let's define the t (a variable indicating time).
+```matlab
+t = 0:0.01:5
+```
+Second, let's define the first sin wave:
+```matlab
+first = 3*sin(2*t+pi/2)
+subplot(2,2,1),plot(t,first)
+title('3sin(2t+pi/2)')
+```
+The subplot allows you to create multiple plots in one figure. The first and the second numbers specify how many rows and columns we want, in our case, it is 2 rows and 2 columns. The last number (1) specify that our plot will be in the first row and the first column (in the upper left corner).
+And the title to define our plot title.
+
+Let's do the same for the second and the third waves.
+```matlab
+second = 3*cos(2*t)
+subplot(2,2,3),plot(t,second)
+title('3cos(2t)')
+
+third = 2*cos(10*t)
+subplot(2,2,[2 4]),plot(t,third)
+xlabel('Time')
+ylabel('Amplitude')
+title('2cos(10t)')
+```
+The `xlabel('')` and the `ylabel('')` defined the names of those labels.
+
+That's all! 
+
+*Note*: please be sure to execute all of the code together to allow the 3 plots to appear in the same figure.
+
+### Conclusion
+This is just a *tip of the iceberg*. Change the amplitude, frequency, phase, subplot numbers, etc.. . Try it out for yourself and see the results. If you are interested more in MATLAB, keep following our [languages](https://www.section.io/engineering-education/topic/languages/) section to create future tutorials about SIMULINK and a lot more. Have a nice day!
```

Comment:
You haven't mentioned anything about amplitude, frequency, and phase in the above section. Most of the readers will not be aware of these concepts. Clarify these before you code it out.